deps/config/default.yml

```diff
@@ -0,0 +1,95 @@
+kind: AKVKeys
```

Comment:
Manifest template files are updated under `./deps/config/manifest/*.template`. Are you intending to change?

Reply:
repinging @nmvalera for this specific topic.

Reply:
Confirmed, I'll remove this.